### PR TITLE
MWPW-141846: remove block wrapper

### PR DESCRIPTION
**Pull request by rl-adobe**
Describe your specific features or fixes

Resolves: [MWPW-141846](https://jira.corp.adobe.com/browse/MWPW-141846)

Test URLs:
- Before: https://main--express--adobecom.hlx.page/express/?martech=off
- After: https://MWPW-141846-remove-block-wrapper--express--adobecom.hlx.page/express/?martech=off

### CODE REVIEW

File: express/blocks/app-banner/app-banner.js [hunk 1]
```diff
@@ -168,7 +168,16 @@ function watchFloatingButtonState(block) {
   }
 }
 
+function addTempWrapper($block, blockName) {
```
Comment (vhargrave) [inline express/blocks/app-banner/app-banner.js:171]: this method can be added as a method to a new file named scripts/decorate.js 
it's used in too many places to redeclare it.

**Review by vhargrave — CHANGES_REQUESTED** [review]
your performance issues I think are being caused by two things: 
1. your test links should have martech (marketing & analytics) turned off by default . You do that by adding ?martech=off as a query param
2. some content is not rendering in the same order anymore. Please do a really thorough test of everything on both desktop & mobile comparing with how things look on production. If you do that you'll start to see issues :)

**Comment by qiyundai** [comment]
@rl-adobe It seems we've got a little conflicts to resolve

**Review by vhargrave — CHANGES_REQUESTED** [review]
You need to merge your change &
I can see content visually breaking on these pages - 

https://mwpw-141846-remove-block-wrapper--express--adobecom.hlx.page/express/learn/blog/30-parenting-blog-topics-and-ideas
https://main--express--adobecom.hlx.page/express/learn/blog/30-parenting-blog-topics-and-ideas
(the table of contents is broken)

And here 
https://mwpw-141846-remove-block-wrapper--express--adobecom.hlx.page/express/create/card
https://main--express--adobecom.hlx.page/express/create/card
The how to steps carousel is just gone. I need you to be really thorough with your testing mate. 

As a tip, just while you're getting into this, you can look for links for each block on this page. Maybe use that to go over every block on mobile and desktop and make sure your code is working as intended . None are just disappearing, they're formatted correctly, etc. 
https://wiki.corp.adobe.com/display/adobedotcom/Express+Blocks+to+be+documented

**Pull request merged**